python/tests/compat/test_mantid.py

```diff
@@ -151,6 +149,27 @@ def test_Workspace2D_not_common_bins_masks(self):
             ds.masks["spectrum"].values[0:3],
             [True, True, True])
 
+    def test_Workspace2D_with_separate_monitors(self):
+        filename = MantidDataHelper.find_file("WISH00016748.raw")
+        ds = mantidcompat.load(filename, LoadMonitors="Separate")
+        monitors = ds.attrs['monitors'].values
+        assert isinstance(monitors, sc.DataArray)
+        assert monitors.shape == [5, 4471]
+
+    def test_Workspace2D_with_include_monitors(self):
+        filename = MantidDataHelper.find_file("WISH00016748.raw")
+        ds = mantidcompat.load(filename, LoadMonitors="Include")
```

Comment:
Should we switch args that are forwarded to Mantid to be a `dict` instead, to avoid future headaches and documentation issues (distinguishing between args scipp knows about and args that scipp forwards)? Something like `mantid_Load_args={'LoadMonitors':'Include'}`?

No other comments, looks good.

Reply:
:+1: I think that would be clearer.